.github/actions/configure_dbt_environment/action.yaml

```diff
@@ -11,16 +11,12 @@ runs:
           echo "On pull request branch, setting dbt env to CI"
           {
             echo "TARGET=ci";
-            echo "CACHE_KEY=pr-caches/$GITHUB_HEAD_REF";
-            echo "CACHE_RESTORE_KEY=master-cache"
```

Comment:
`CACHE_RESTORE_KEY` is no longer necessary since we're not using fallback caches anymore.